src/lib/make-toolbox-xml.js

```diff
@@ -247,6 +244,8 @@ const looks = function (isStage, targetId) {
                 </value>
             </block>
             ${blockSeparator}
+        <block type="looks_show"/>
```

Comment:
These should go above the `looks_gotofrontback` block to maintain the current block ordering. Make sure to keep the a block separator between the show/hide and the layering blocks. 

Also, the indentation is wrong, it should be aligned with the `looks_gotofrontback` block since it is part of the same  indentation level.

Reply:
Okay, I changed it.